metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -63,8 +76,50 @@ def get_session_and_host():
 
             gms_host = gms_config.server
             gms_token = gms_config.token
+            return gms_host, gms_token
         except yaml.YAMLError as exc:
             click.secho(f"{DATAHUB_CONFIG_PATH} malformatted, error: {exc}", bold=True)
+    return None, None
+
+
+def get_details_from_env() -> Tuple[Optional[str], Optional[str]]:
+    gms_host: Optional[str] = None
+    try:
+        gms_host = os.environ[ENV_METADATA_HOST]
+    except KeyError:
+        pass
+
+    gms_token: Optional[str] = None
+    try:
+        gms_token = os.environ[ENV_METADATA_TOKEN]
+    except KeyError:
+        pass
+
+    return gms_host, gms_token
+
+
+def first_non_null(ls: List[Optional[str]]) -> Optional[str]:
+    return next((el for el in ls if el is not None), None)
```

Comment:
should this also skip on `""` since we consider that an error below?

Reply:
good point